provider/resources.go

```diff
@@ -189,21 +201,25 @@ func Provider() tfbridge.ProviderInfo {
 			PackageReferences: map[string]string{
 				"Pulumi": "3.*",
 			},
-		},
+		}, MetadataInfo: tfbridge.NewProviderMetadata(metadata),
 	}
 
 	err := x.ComputeDefaults(&prov, x.TokensSingleModule("pagerduty_",
 		mainMod, x.MakeStandardToken(mainPkg)))
-	contract.AssertNoError(err)
+	contract.AssertNoErrorf(err, "computing defaults failed")
+	err = x.AutoAliasing(&prov, prov.GetMetadata())
+	contract.
+
+		// CustomField.name can only contain 'a'-'z' '0'-'9' and '_'.
+		//
+		// Non-conforming runes are replaced with '_'.
```

Comment:
@aq17 I think we need to figure out how to insert AST nodes without disturbing existing comments.

Reply:
Yep 😕